payments.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
```

Comment:
I would prefer naming it "default-payment-config" or similar, thus providing more context.